crs/v1alpha2/controlplane_ubuntu.yaml

```diff
@@ -96,3 +114,16 @@ spec:
         owner: root:root
         permissions: '0644'
         content: ${SSH_PUB_KEY_CONTENT}
+      - path : /etc/netplan/60-ironicendpoint.yaml
+        owner: root:root
+        permissions: '0644'
+        content: |
+          network:
+            version: 2
+            renderer: networkd
+            bridges:
+              ironicendpoint:
+                interfaces: [enp1s0]
+                dhcp4: no
```

Comment:
here should probably be dhcp

Reply:
No. dhcp would assign higher IP to this interface and then the keepalived for ironic endpoint IP will be taken over. We cannot afford this before pivoting

Reply:
you don't have keepalived running before you do the pivoting, so it does not matter, does it ?

Reply:
keepalived runs before pivoting ... since it assigns the APIEndPoint IP also ...

Reply:
I think I understand now what you mean and you have a point. I will change and check how it goes.

Reply:
@maelk Done